### ISSUE TITLE
Drop $web$ and $test$ placeholders

### ISSUE THREAD

**Issue by natebosch** [dart-lang/build#2575]
`$package$` should be used instead.

**Comment by daniel-v** [comment]
When that transition happens, please make sure to add appropriate documentation or changelog entries so that we have somewhere to look when migrating our builders. Thank you

**Comment by jakemac53** [comment]
Yes we definitely will when we do drop these. You should be able to get ahead of this and migrate to `$package$` at any time though, and if that causes a problem for you it would be good to know early as well (for us).

**Comment by daniel-v** [comment]
I'd love to give you feedback on it and willing to migrate all my builders to it as a test but I am stuck at step 1: I have no idea what $package$ means. Could you point me to the right place where I could gather some info on it?

**Comment by jakemac53** [comment]
Are you familiar with `$test$` and `$web$`? If so, `$package$` is the same except that it lives at the root of the package, while those live under their corresponding subdirectories (`test/$test$`, `web/$web$` etc).

**Comment by daniel-v** [comment]
Quick prototype shows, that our builder works with `$package$` just as well as with `$web$`. Obviously, paths had to be updated, since assets are located 1 level deeper in the directory structure but other than that, it seems to be OK at first glance.

**Comment by davidmorgan** [comment]
Closing as stale.
